crates/solvers/src/tests/legacy/jit_order.rs

```diff
@@ -2,13 +2,20 @@
 //! serialized.
 
 use {
-    crate::tests::{self, legacy},
+    crate::tests::{self, legacy, mock},
     serde_json::json,
 };
 
 #[tokio::test]
 async fn test() {
-    let legacy_solver = tests::legacy::setup(vec![legacy::Expectation {
+    let legacy_solver = mock::http::setup(vec![mock::http::Expectation::Post {
+        path: mock::http::Path::glob(
+            "solve\
+             [?]instance_name=*_Mainnet_1_0\
+               &time_limit=*\
+               &max_nr_exec_orders=100\
+               &use_internal_buffers=true"
```

Comment:
I think it would be cool if we could reduce the test code to the things that matter. As long as we have 1 test that covers the query parameters I think adding this glob path to all the tests makes them unnecessarily noisy.

Reply:
Good point. I changed the tests to so that only the `market_orders` test verifies the query string (one for `quote` and one for `solve`).

da21fd61d6d8001ec8c06b093e7ae10b0b0c6af7